### PR TITLE
nroam-backlinks.el: Fix edge case (again)

### PR DESCRIPTION
**Pull request by Whil-**
* nroam-backlinks.el (nroam-backlinks--crawl-source): Set full-outline
to nil if point is before first heading using a built in org function.

### CODE REVIEW

**Comment by NicolasPetton** [comment]
Awesome, thanks!

File: nroam-backlinks.el [hunk 1]
```diff
@@ -120,12 +120,13 @@ When nil, LEVEL defaults to 3."
   (with-current-buffer (nroam-backlinks--work-buffer)
     (insert-file-contents file nil nil nil 'replace)
     (goto-char point)
-    (let* ((elt (org-element-at-point))
-           (begin (org-element-property :begin elt))
-           (end (org-element-property :end elt))
-           (type (org-element-type elt))
-           (outline (org-get-outline-path))
-           (full-outline (and outline (org-get-outline-path 'with-self))))
+    (let ((elt (org-element-at-point))
```
Comment (NicolasPetton) [inline nroam-backlinks.el:123]: Why replace `let*` with `let`? `elt` is used below in other let forms, so this breaks.
FWY I just fixed it in master.

**Comment by Whil-** [comment]
Seems I messed up though! New pull request coming soon. I shouldn't have switched from `let*` to `let`!

**Comment by NicolasPetton** [comment]
> Seems I messed up though! New pull request coming soon. I shouldn't have switched from `let*` to `let`!

I already fixed it :)

File: nroam-backlinks.el (hunk 1 above)
Comment (Whil-) [reply nroam-backlinks.el:123]: Ah, ok. Thanks. Yeah, as mentioned a few seconds ago in the comment above I messed up.

But then I know it's fixed :)

**Pull request merged**